tests/test_constants.py

```diff
@@ -1,8 +1,11 @@
 import inspect
 import typing
 import unittest
+from dotenv import load_dotenv
 from bot import constants
 
+load_dotenv()
```

Comment:
Was this test failing without dotenv?

Reply:
test discovery was failing completely without this load. but I think it might have been a knock-on from another error (rather than being caused by a missing load_dotenv), since I removed these lines and discovery is still working. removing these changes for now.